dev-docs/bidders/essens.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: bidder
-title: AdBund
-description: Prebid AdBund Bidder Adaptor
+title: Eseens
```

Comment:
Hey @navneetpandey I think this might be a typo:  Should it be changed from 'Eseens' to 'Essens' ?

Reply:
Yes please. Thanks .